bucket/shadowsocks.json

```diff
@@ -27,12 +27,15 @@
         "pac.txt",
         "user-rule.txt"
     ],
-    "checkver": "github",
+    "checkver": {
+        "url": "https://api.github.com/repos/shadowsocks/shadowsocks-windows/releases/latest",
+        "regex": "(?s)\"tag_name\":\"([\\d.]+)\".+Shadowsocks-(?<main>[\\d.]+)\\.(?<build>\\d+).zip"
+    },
     "autoupdate": {
-        "url": "https://github.com/shadowsocks/shadowsocks-windows/releases/download/$version/Shadowsocks-$version.zip",
+        "url": "https://github.com/shadowsocks/shadowsocks-windows/releases/download/$version/Shadowsocks-$matchMain.$matchBuild.zip",
```

Comment:
```suggestion
        "url": "https://github.com/shadowsocks/shadowsocks-windows/releases/download/$matchHead.0/Shadowsocks-$version.zip",
```

Reply:
Well, after checking the tag list, I don't think that we can assume that the last number is always 0.

Reply:
I can. It is broken since version 4.3.0.0. After that there were no different naming.